setup.cfg

```diff
@@ -1,4 +1,5 @@
 [pycodestyle]
+ignore = W602
```

Comment:
Ya, I tried that too... [noqa only disables some rules](http://pycodestyle.pycqa.org/en/latest/intro.html#configuration) and W602 isn't one of them.